pkg/apps/srv/backstage/install.yaml

```diff
@@ -116,17 +152,17 @@ data:
   app-config.yaml: |
     app:
       title: CNOE
-      baseUrl: http://localhost:3000
+      baseUrl: "https://backstage.idpbuilder.cnoe.io.local:8443"
```

Comment:
These needs to be either `local.me` or `local.cnoe.io`, depending on which one we use in #29

Reply:
I suggest to change all the URLs (including what we have here + documentation when a PR will be created to fix issue 29

Reply:
Yep - we're going to hopefully merge https://github.com/cnoe-io/idpbuilder/pull/29 asap which does this, we should change it here assuming we merge that one first.

Reply:
I updated the DNS name to use cnoe.io.localtest.me @greghaynes